src/app/components/SliderComponent/SliderComponent.tsx

```diff
@@ -19,83 +21,6 @@ export const SliderComponent: React.FC<SliderComponentProps> = ({
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const [swiper, setSwiper] = useState<any | null>(null);
```

Comment:
Musi byc any?

Reply:
Nie dotykalam logiki w tym komponencie, ok, ja to wyprawie

Reply:
zajmuję się tym :)